oz/core/actions.py

```diff
@@ -65,6 +65,13 @@ def init(project_name):
 def server():
     """Runs the server"""
 
+    if oz.settings["logging"]:
```

Comment:
that should be a separate thing. `enable_pretty_logging` should just enable ANSI colors in the terminal.